src/include/txn.i

```diff
@@ -9,6 +9,12 @@
 static inline int __wt_txn_id_check(WT_SESSION_IMPL *session);
 static inline void __wt_txn_read_last(WT_SESSION_IMPL *session);
 
+typedef enum {
+	WT_VISIBLE_TYPE_FALSE=0,        /* Not a visible update */
+	WT_VISIBLE_TYPE_PREPARE=1,      /* Prepared update */
+	WT_VISIBLE_TYPE_TRUE=2          /* A visible update */
+} WT_VISIBLE_TYPE;
```

Comment:
I'd change the constant names to `WT_VISIBLE_FALSE`, etc. (remove `_TYPE`) to keep them a little shorter.